ogusa/tests/test_get_micro_data.py

```diff
@@ -197,7 +197,6 @@ def test_taxcalc_advance():
         assert np.allclose(expected_dict[k], v, equal_nan=True)
 
 
-@pytest.mark.full_run
 def test_cap_inc_mtr():
```

Comment:
@jdebacker. Are you adding this `test_cap_inc_mtr()` test to the GH Actions tests that run on every commit to a PR? Do we have room for this in our computation time allotment?

Reply:
This test only takes a minute. I'm not sure why it was a `full_run` test before - perhaps because it having used the PUF in the past?